src/client/java/teammates/client/scripts/RepairFeedbackSessionNameWithExtraWhiteSpace.java

```diff
@@ -0,0 +1,147 @@
+        migrator.doOperationRemotely();
+    }
+
+    @SuppressWarnings("deprecation")
+    @Override
+    protected void doOperation() {
+        Datastore.initialize();
+        
+        List<FeedbackSession> feedbackSessionList = feedbackSessionsDb.getAllFeedbackSessionEntities();
+        System.out.println("There is/are " + feedbackSessionList.size() + " session(s).");
+        
+        if (isPreview) {
+            System.out.println("Checking extra spaces in feedback session name...");
+        } else {
+            System.out.println("Removing extra spaces in feedback session name...");
+        }
+        
+        try {
+            int numberOfFeedbackSessionWithExtraWhiteSpacesInName = 0;
+            for (FeedbackSession session : feedbackSessionList) {
+                if (hasExtraSpaces(session.getFeedbackSessionName())) {
+                    numberOfFeedbackSessionWithExtraWhiteSpacesInName++;
+                    if (isPreview) {
+                        showFeedbackSession(session);
+                    } else {
+                        fixFeedbackSession(session);
+                    }
+                }
+            }
+            
+            if (isPreview) {
+                System.out.println("There are/is " + numberOfFeedbackSessionWithExtraWhiteSpacesInName
+                                   + "/" + feedbackSessionList.size() + " feedback session(s) with extra spaces in name!");
+            } else {
+                System.out.println("" + numberOfFeedbackSessionWithExtraWhiteSpacesInName
+                                   + "/" + feedbackSessionList.size() + " feedback session(s) have been fixed!");
+                System.out.println("Extra space removing done!");
+            }
+        } catch (InvalidParametersException | EntityDoesNotExistException | EntityAlreadyExistsException e) {
+            e.printStackTrace();
+        }
+    }
+    
+    /**
+     * Displays the feedback session.
+     */
+    private void showFeedbackSession(FeedbackSession session) {
+        System.out.println("Feedback Session Name: \"" + session.getFeedbackSessionName() + "\" in " + session.getCourseId());
+    }
+
+    /**
+     * Remove extra spaces in feedback session's name
+     */
+    private void fixFeedbackSession(FeedbackSession session) throws InvalidParametersException, EntityDoesNotExistException, EntityAlreadyExistsException {
+        fixFeedbackQuestionsOfFeedbackSession(session);
+        fixFeedbackResponsesOfFeedbackSession(session);
+        fixFeedbackResponseCommentsOfFeedbackSession(session);
+        
+        FeedbackSessionAttributes sessionAttribute = new FeedbackSessionAttributes(session);
+        feedbackSessionsDb.deleteEntity(sessionAttribute);
+        sessionAttribute.feedbackSessionName = StringHelper.removeExtraSpace(sessionAttribute.getFeedbackSessionName());
+        feedbackSessionsDb.createEntity(sessionAttribute);
```

Comment:
What I did before was `session.setFeedbackSessionName(StringHelper.removeExtraSpace(session.getFeedbackSessionName()))` but I was not sure if it would work or not.